### ISSUE TITLE
Some cognate class views aren't working

### ISSUE THREAD

**Issue by m-scarborough** [lingdb/CoBL#237]
This is not consistent overall:

For example in 'meat' class C http://cobl.info/cognate/437/ works, but class I http://cobl.info/cognate/1508/ returns the standard 'page unavailable' error message.

**Comment by runjak** [comment]
Interesting:
The example link for `page unavailable` appears to be broken on the server but not on my machine.

**Comment by runjak** [comment]
If I'm correct it appears that I introduced a small problem when working on #229.
![screenshot_2016-07-25_14-24-41](https://cloud.githubusercontent.com/assets/5417642/17101208/8c42aa56-5273-11e6-9440-450e760c9332.png)

The reliability should always be one that is defined in [models.py:50-56](https://github.com/lingdb/CoBL/blob/87f891e6f5bbea3866f5c01e7db947289551ac4f/ielex/lexicon/models.py#L50-L56):
![screenshot_2016-07-25_14-27-00](https://cloud.githubusercontent.com/assets/5417642/17101293/06da9bb6-5274-11e6-8b82-f07eabc98a3e.png)

But a few entries don't follow this rule at the moment:
![screenshot_2016-07-25_14-26-07](https://cloud.githubusercontent.com/assets/5417642/17101300/149b22d4-5274-11e6-95c1-9b01fb598b95.png)

My fix will consist of these steps:

* Produce a list of affected entries so that it's possible to track what is affected.
* Set a default reliability to make sure we avoid this inconsistency problem in the future.
* Fix the incorrect database entries so that the sites work again.
* Expand error reporting on the server side a bit so that I can figure out similar problems faster in the future.

**Comment by runjak** [comment]
The ids of offending entries in the database are:
[offending.txt](https://github.com/lingdb/CoBL/files/381610/offending.txt)

I'll compare these against different backups to see if they had correct reliabilities before.

Also I have a fix that should make it possible to view the problematic pages again that I will deploy soon.

**Comment by runjak** [comment]
Looking at the offending ids I think that the offending entries are likely to have been handled in bulk. Except for the first id they are quite close together:
![screenshot_2016-07-25_15-43-09](https://cloud.githubusercontent.com/assets/5417642/17103474/b39fba02-527e-11e6-831f-46295a9bd85c.png)

**Comment by runjak** [comment]
In the backup from `2013-07-23` the number of offending ids is much smaller:
```python
(44110, u'')
(46480, u'')
(46481, u'')
(46482, u'')
(46483, u'')
(46484, u'')
(46485, u'')
(46486, u'')
(46487, u'')
(46488, u'')
(46489, u'')
(46490, u'')
(46491, u'')
(46492, u'')
(46493, u'')
(46494, u'')
```
The backups from `2016-07-21`, `2016-07-21`, `2016-07-15`, `2016-07-13` and `2016-07-01` have only one entry:
```python
(44110, u'')
```

**Comment by runjak** [comment]
I'll let the site display the entries without a reliability rating like the lower ones in this table:
![screenshot_2016-07-25_16-50-01](https://cloud.githubusercontent.com/assets/5417642/17105542/ff0e9d06-5287-11e6-874f-fa9285c2ac43.png)

**Comment by runjak** [comment]
* I've just deployed a fix for this on the server, so that http://cobl.info/cognate/1508/ can be viewed again.
* I've also improved the current logging mechanisms with 5b608e6 and 87f891e so that it should be easier for me to get detailed information on server side problems when something is broken.
* A list of `CognateJudgementCitation` entries that have a reliability of `''` was created, but I'm not sure what to do with it. It appears as if these entries didn't have a reliability set at any point in the past that I have backups for. Together with the fact that their ids are close together I think it could be that they have been generated automatically. It would probably be helpful for me to discuss this with @PaulHeggarty or @cormacanderson tomorrow. Given that #229 asks for these reliabilities to be disabled in the future anyway it may also be that it's not worth bothering about the current situation for the odd entries.
